zhaquirks/tuya/neo_TS0201.py

```diff
@@ -0,0 +1,124 @@
+    cluster_id = 0xE002
+
+    manufacturer_attributes = {
+        # Alarm settings
+        0xD00A: ("alarm_temperature_man", t.uint16_t),
+        0xD00B: ("alarm_temperature_min", t.uint16_t),
+        0xD00C: ("alarm_humidity_max", t.uint16_t),
+        0xD00E: ("alarm_humidity_min", t.uint16_t),
+        # Alarm information
+        0xD00F: ("alarm_humidity", ValueAlarm),
+        0xD006: ("temperature_humidity", ValueAlarm),
+        # Unknown
+        0xD010: {"unknown", t.uint8_t},
+    }
+
+
+class NeoCustomTemperatureHumidityEndpoint(Endpoint):
+    """Neo custom temperature and humidity endpoint"""
+
+    def __init__(self, device, endpoint_id):
+        super().__init__(device, endpoint_id)
+        self.info("Forcing discovery information for Neo custom endpoint")
+
+        self.profile_id = zha.PROFILE_ID
+        self.device_type = DeviceType.TEMPERATURE_SENSOR
+        self.device_type = zigpy.profiles.zha.DeviceType(self.device_type)
+
+        self.add_input_cluster(
+            TemperatureMeasurement.cluster_id,
+            TemperatureMeasurement(endpoint=self, is_server=True),
+        )
+        self.add_input_cluster(
+            RelativeHumidity.cluster_id,
+            RelativeHumidity(endpoint=self, is_server=True),
+        )
+        self.status = zigpy.endpoint.Status.ZDO_INIT
+
+
+class TemperatureHumidtyIlluminanceSensor(CustomDevice):
+    """Neo Tuya Temperature, Humidity and Illumination Sensor."""
+
+    def __init__(self, *args, **kwargs):
+        """Init."""
+        super().__init__(*args, **kwargs)
+        # Add missing endpoint 2
+        self.endpoints[2] = NeoCustomTemperatureHumidityEndpoint(self, 2)
+
+    signature = {
+        #  <SimpleDescriptor endpoint=1, profile=260, device_type=262
+        #  device_version=1
+        #  input_clusters=[0, 1, 1024, 57346]
+        #  output_clusters=[25, 10]>
+        MODELS_INFO: [("_TZ3000_qaaysllp", "TS0201")],
+        ENDPOINTS: {
+            1: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: DeviceType.LIGHT_SENSOR,
+                INPUT_CLUSTERS: [
+                    Basic.cluster_id,
+                    PowerConfiguration.cluster_id,
+                    IlluminanceMeasurement.cluster_id,
+                    NeoTemperatureHumidityAlarmCluster.cluster_id,
+                ],
+                OUTPUT_CLUSTERS: [
+                    Time.cluster_id,
+                    Ota.cluster_id,
+                ],
+            },
+        },
+    }
+
+    replacement = {
+        ENDPOINTS: {
+            1: {
+                INPUT_CLUSTERS: [
+                    Basic.cluster_id,
+                    PowerConfiguration.cluster_id,
+                    IlluminanceMeasurement.cluster_id,
+                    NeoTemperatureHumidityAlarmCluster.cluster_id,
+                ],
+                OUTPUT_CLUSTERS: [
+                    Time.cluster_id,
+                    Ota.cluster_id,
+                ],
+            },
```

Comment:
```suggestion
            },
            2: {
                     PROFILE_ID: zha.PROFILE_ID,
                     DEVICE_TYPE: DeviceType.LIGHT_SENSOR,
                     INPUT_CLUSTERS: [
                         Basic.cluster_id,
                         TemperatureMeasurement.cluster_id,
                         RelativeHumidity.cluster_id,  
                     ],                   
            }
```

Reply:
awesome! I will try it out and report back